packages/components/tsconfig.json

```diff
@@ -5,7 +5,6 @@
     "compilerOptions": {
         "paths": {
             "@hopper-ui/styled-system": ["../styled-system/src/index.ts"],
-            "@hopper-ui/components": ["../components/src/index.ts"],
```

Comment:
just curious about the reason why this was removed?

Reply:
inside @hopper-ui/components, it doesn't make much sense to import things from our own package

Reply:
This was what was causing an issue with the exports

Reply:
ohh, I get it now.
but there must be a reason it was added before.
This might be a stupid question, but how do we know @hopper-ui/components links to "../components/src/index.ts"

Reply:
Not sure i 100% understand the question, so if my answer is off let me know. 

If you are asking "how do we know @hopper-ui/components links to "../components/src/index.ts"" for our consumers, that happens in the package.json. The name of course, and the files it redirects to are detailled in the exports section. 

In the TSConfig, it's only for our  own code that references other packages. For instance, when we reference @hopper-ui/styled-system in the components package, we want to be able to ctrl+click something we import and navigate to the src code.  That is the purpose of this line. Normally, ctrl+clicking would point to the /dist folder of the styled-system, AND we would need to build all the time to get the latest updates between the packages

Reply:
right, yes this answers everything, thanks!